File: Python/Processor.py
# membuat class
class Processor():

	# deklarasi atribut private
	__name = ""
	__price = 0

	# constructor
	def __init__(self):
		self.__name = ""
		self.__price = 0

	# setter dan getter untuk atribut name
	def setName(self, name):
		self.__name = name

	def getName(self):
		return self.__name;
	
	# setter dan getter atribut price 
	def setPrice(self, price):
		self.__price = price

	def getPrice(self):
		return self.__price

	# method untuk mencetak output atribut dari class 
	def printProcessor(self):
		print("Processor   : " + self.__name)
		print("Harga       : Rp{}".format(self.__price))
		print("---------------------------------------")

File: PHP/Processor.php
<?php

/* membuat class */ 
class Processor{

/* deklarasi atribut private */
	private $name = "";
	private $price = 0;

	/* constructor */
	public function __construct($name = "", $price = 0){
		$this->name = $name;
		$this->price = $price;
	}

	/* setter dan getter untuk atribut name */
	public function setName($name){ 
		$this->name = $name;
	}

	public function getName(){
		return $this->name;
	}

	/* setter dan getter atribut price */
	public function setPrice($price){ 
		$this->price = $price;
	}

	public function getPrice(){
		return $this->price;
	}

	/* method untuk mencetak output atribut dari class */ 
	public function printProcessor(){
		echo "Processor   : " . $this->name . "<br>";
		echo "Harga       : Rp" . $this->price . "<br>";
		echo "------------------------------------------------------<br>";
	}
}

?>